sql/src/test/java/io/crate/execution/engine/collect/stats/JobsLogsTest.java

```diff
@@ -87,25 +87,26 @@ public void terminateScheduler() throws InterruptedException {
     @Test
     public void testDefaultSettings() {
         JobsLogService stats = new JobsLogService(Settings.EMPTY, clusterSettings, scheduler, breakerService);
-        assertThat(stats.isEnabled(), is(false));
+        assertThat(stats.isEnabled(), is(true));
         assertThat(stats.jobsLogSize, is(JobsLogService.STATS_JOBS_LOG_SIZE_SETTING.getDefault()));
         assertThat(stats.operationsLogSize, is(JobsLogService.STATS_OPERATIONS_LOG_SIZE_SETTING.getDefault()));
         assertThat(stats.jobsLogExpiration, is(JobsLogService.STATS_JOBS_LOG_EXPIRATION_SETTING.getDefault()));
         assertThat(stats.operationsLogExpiration, is(JobsLogService.STATS_OPERATIONS_LOG_EXPIRATION_SETTING.getDefault()));
-
-        // even though jobsLog size and opertionsLog size are > 0 it must be a NoopQueue because the stats are disabled
-        assertThat(stats.jobsLogSink, Matchers.instanceOf(NoopLogSink.class));
-        assertThat(stats.operationsLogSink, Matchers.instanceOf(NoopLogSink.class));
     }
 
     @Test
-    public void testEnableStats() throws Exception {
+    public void testEnableStats() {
```

Comment:
rename to `testReEnableStats()`
because it assumes that the stats were disabled before